packages/admin-portal/src/actions/subscriptions.ts

```diff
@@ -0,0 +1,14 @@
+import { actionCreator } from '../utils/actions'
+import ActionTypes from '../constants/action-types'
+import { SubscriptionModelPagedResult } from '@reapit/foundations-ts-definitions'
+
+export interface fetchSubscriptionListValues {
+  page: number
+  queryString: string
+}
+
+export const fetchSubscriptionList = actionCreator<fetchSubscriptionListValues>(ActionTypes.FETCH_SUBCRIPTION_LIST)
```

Comment:
Just dispatch this action when you click on a developer in your developer drop down list